### ISSUE TITLE
Suggestion: outdoor milking/fleshlight stations

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#130]
It becomes very inconvenient to use the milking and fleshlight stations in the kitchen when the kobold you want to milk is just about too tall to squeeze into the kitchen or the kobold you want to pump is too long of shaft to successfully line her dick up into the machine. My prize kobold is too cumbersome to play as for much of the time, so I'd like to keep her busy refilling my watering cans with cum.

Suppose that the construction shop's upgrade options continue into the teens. Two options you might provide at this tier are a pair of outdoor duplicates of these stations, providing the dual benefits of having another such station for harvesting and being able to keep up with the size amplifications encouraged by the quest system.

**Comment by Godeken** [comment]
I dont speak for Naelstrof, but what i have seen - the game uses an animation-station type of system which is really cool. It allows one to create an animation of the character doing something and allows you to add timing in the background to process something. This is 100x easier to add animations and modify actions that ibdividually programming each and every one and creating seperate animations for each... (Trying to make it sound as basic as possible)

The downside, is that we have the scale issues when a Kobold is too big or too small... a large Kobold kinda does the Macarena when trying turn the grinder, or the boobs do the YMCA movement when giving birth on one of the stands.

**Comment by naelstrof** [comment]
Yeah there needs to be more stations in general. It'd be nice to accommodate macro kobolds better too--
